.github/ISSUE_TEMPLATE/feature-development.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature development
+about: Feature currently being developed / reviewed
+title: "[FEATURE]"
+labels: enhancement
+assignees: ''
```

Comment:
Shouldn't we add milestone here? And maybe default it to `Certifiable controller on Ubuntu NUC` (the milestone I added already).